i18n/locales/whitelist_fr.json

```diff
@@ -0,0 +1,2 @@
+[
+]
```

Comment:
I'm not 100% on the usage of this, but it's a result of `react-intl-translations-manager`